src/das-employer-favourites.ApplicationServices/Configuration/ConnectionStrings.cs

```diff
@@ -0,0 +1,8 @@
+namespace DfE.EmployerFavourites.ApplicationServices.Configuration
```

Comment:
If we're following the onion architecture layering this should live in Infrastructure